pyproject.toml

```diff
@@ -89,3 +89,6 @@ enable_error_code = [
 ]
 show_error_codes = true
 warn_unreachable = true
+
+[project.entry-points."beamlime.stateless"]
+dummy = "beamlime.stateless_workflow:DummyWorkflow"
```

Comment:
You probably need to change this now, e.g., to a function that returns and instance. Did you try running `beamlime`, to see if it still works after your latest changes?

Reply:
No I haven't opened the repo yet ;) but I'll make sure to test it before merge